<ui version="4.0" >
 <class>IncludeUi</class>
 <widget class="QDialog" name="IncludeUi" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string/>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="3" column="0" colspan="5" >
    <widget class="QGroupBox" name="listingsGB" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>7</hsizetype>
       <vsizetype>7</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>Listing Parameters</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="3" column="1" >
       <widget class="QTextEdit" name="listingsED" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>5</hsizetype>
          <vsizetype>7</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2" >
       <widget class="QCheckBox" name="bypassCB" >
        <property name="toolTip" >
         <string>Check it to enter parameters that are not recognizable by LyX</string>
        </property>
        <property name="text" >
         <string>&amp;Bypass validation</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLineEdit" name="captionLE" >
        <property name="minimumSize" >
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="captionLabel" >
        <property name="text" >
         <string>C&amp;aption:</string>
        </property>
        <property name="buddy" >
         <cstring>captionLE</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLineEdit" name="labelLE" />
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="labelLabel" >
        <property name="text" >
         <string>La&amp;bel:</string>
        </property>
        <property name="buddy" >
         <cstring>labelLE</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>Mo&amp;re parameters</string>
        </property>
        <property name="buddy" >
         <cstring>listingsED</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QTextBrowser" name="listingsTB" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>0</hsizetype>
          <vsizetype>7</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize" >
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="cursor" >
         <cursor>0</cursor>
        </property>
        <property name="focusPolicy" >
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="acceptDrops" >
         <bool>true</bool>
        </property>
        <property name="frameShape" >
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth" >
         <number>0</number>
        </property>
        <property name="acceptRichText" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="5" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QCheckBox" name="visiblespaceCB" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>3</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip" >
        <string>Underline spaces in generated output</string>
       </property>
       <property name="text" >
        <string>&amp;Mark spaces in output</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="previewCB" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>3</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip" >
        <string>Show LaTeX preview</string>
       </property>
       <property name="text" >
        <string>&amp;Show preview</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0" colspan="5" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okPB" >
       <property name="text" >
        <string>&amp;OK</string>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closePB" >
       <property name="text" >
        <string>&amp;Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" >
    <widget class="QLabel" name="filenameLA" >
     <property name="toolTip" >
      <string/>
     </property>
     <property name="text" >
      <string>&amp;File:</string>
     </property>
     <property name="buddy" >
      <cstring>filenameED</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3" >
    <widget class="QLineEdit" name="filenameED" >
     <property name="toolTip" >
      <string>File name to include</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2" >
    <widget class="QLabel" name="TextLabel1" >
     <property name="text" >
      <string>&amp;Include Type:</string>
     </property>
     <property name="buddy" >
      <cstring>typeCO</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="2" >
    <widget class="QComboBox" name="typeCO" >
     <item>
      <property name="text" >
       <string>Include</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Input</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Verbatim</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Program Listing</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="3" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>28</width>
       <height>28</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="4" >
    <widget class="QPushButton" name="browsePB" >
     <property name="toolTip" >
      <string>Select a file</string>
     </property>
     <property name="text" >
      <string>&amp;Browse...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4" >
    <widget class="QPushButton" name="editPB" >
     <property name="toolTip" >
      <string>Edit the file</string>
     </property>
     <property name="text" >
      <string>&amp;Edit</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>filenameED</tabstop>
  <tabstop>browsePB</tabstop>
  <tabstop>typeCO</tabstop>
  <tabstop>editPB</tabstop>
  <tabstop>visiblespaceCB</tabstop>
  <tabstop>previewCB</tabstop>
  <tabstop>captionLE</tabstop>
  <tabstop>labelLE</tabstop>
  <tabstop>listingsTB</tabstop>
  <tabstop>listingsED</tabstop>
  <tabstop>okPB</tabstop>
  <tabstop>closePB</tabstop>
 </tabstops>
 <includes>
  <include location="local" >qt_helpers.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
